doc/nrf/app_dev/device_guides/nrf54l/snippets.rst

```diff
@@ -0,0 +1,92 @@
+.. _nRF54l_snippets:
+
+Snippets for nRF54L05 and nRF54L10
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can emulate the nRF54L05 and nRF54L10 on an nRF54L15 device using the following snippets:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Functionality
+     - Snippet name
+     - Compatible board targets
+   * - :ref:`emulated-nrf54l05`
+     - ``emulated-nrf54l05``
+     - ``nrf54l15dk/nrf54l15/cpuapp``, ``nrf54l15dk/nrf54l15/cpuapp/ns``
+   * - :ref:`emulated-nrf54l10`
+     - ``emulated-nrf54l10``
+     - ``nrf54l15dk/nrf54l15/cpuapp``, ``nrf54l15dk/nrf54l15/cpuapp/ns``
+
+.. warning::
+   These snippets cannot be used with the FLPR core because all memory, including RAM and RRAM, is allocated to the application core.
```

Comment:
```suggestion
.. important::
   You cannot use these snippets with the FLPR core because all memory, including RAM and RRAM, is allocated to the application core.
```
warning is too strong, if important is too weak, maybe caution